.github/ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,6 @@
+## Deprecation warning ⚠️
+
+The Timber theme is no longer being maintained by Shopify.  Issues will not be addressed by this
```

Comment:
Me and Obama. https://www.reddit.com/r/CGPGrey2/comments/5p7p64/til_president_obama_also_uses_double_spaces_after/